hr_loan_management/data/base_action_rule_data.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 OpenSynergy Indonesia
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
+
+<record id="base_action_rule_01" model="base.action.rule">
+    <field name="name">Employee Loan Realization</field>
+    <field name="model_id" ref="account.model_account_move_line"/>
+    <field name="sequence" eval="1"/>
+    <field name="kind">on_write</field>
+    <field name="server_action_ids" eval="[(4, ref('hr_loan_management.ir_action_server_01'))]"/>
+    <field name="filter_id" ref="hr_loan_management.ir_filter_04"/>
+</record>
+
+<record id="base_action_rule_02" model="base.action.rule">
+    <field name="name">Employee Loan Payment</field>
+    <field name="model_id" ref="account.model_account_move_line"/>
+    <field name="sequence" eval="1"/>
+    <field name="kind">on_write</field>
+    <field name="server_action_ids" eval="[(4, ref('hr_loan_management.ir_action_server_02'))]"/>
+    <field name="filter_id" ref="hr_loan_management.ir_filter_04"/>
+</record>
```

Comment:
Well i need functionality like this [transition ](https://github.com/odoo/odoo/blob/8.0/addons/account/account_invoice_workflow.xml#L88). Since workflow might be fully depreceted on later version of odoo, my options are:

1. Code directly on account.move.line. 
2. Automated actions

I like options 2 better since due it's low risk conflicting with other module. Suggestions are most welcome though.